pom.xml

```diff
@@ -1758,12 +1769,26 @@
                 <version>${kafka.version}</version>
             </dependency>
             <!-- //Kafka -->
-            <!-- OpenTracing -->
+            <!-- Open Telemetry -->
             <dependency>
-                <groupId>io.opentracing</groupId>
-                <artifactId>opentracing-mock</artifactId>
-                <version>${opentracing.mock.version}</version>
-                <scope>test</scope>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-sdk-extension-autoconfigure-spi</artifactId>
+                <version>${opentelemetry.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.opentelemetry.instrumentation</groupId>
+                <artifactId>opentelemetry-instrumentation-api</artifactId>
+                <version>${opentelemetry.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.opentelemetry.javaagent</groupId>
+                <artifactId>opentelemetry-javaagent-extension-api</artifactId>
+                <version>${opentelemetry.version}-alpha</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.auto.service</groupId>
+                <artifactId>auto-service</artifactId>
+                <version>${auto.service.version}</version>
```

Comment:
For which purposes is this new dependency needed? Can we remove it?

Reply:
According to the https://github.com/google/auto/tree/main/service it is supplementary for development only and does not required for OpenTelementry neither compile time nor runtime.